rust-toolchain

```diff
@@ -1 +1,11 @@
-1.50.0
+[toolchain]
+channel = "1.50.0"
+targets = [
+    "aarch64-linux-android",
+    "armv7-linux-androideabi",
+    "i686-linux-android",
+    "x86_64-linux-android", 
+    "aarch64-apple-ios",
+    "x86_64-apple-ios"
+]
+components = ["clippy", "rustfmt"]
```

Comment:
Both CI and local developers will want these components installed, let's make them happen by default.